packages/ui-library/src/components/form-caption/index.stories.ts

```diff
@@ -73,16 +73,19 @@ export default {
       description: {
         component: `<Markdown>
         Form Caption provides either a brief hint or error message to a related form element such as Input Field Text or Select. It is typically displayed below the form element, and it can be used to provide additional information about the type of information a user has to provide, like an example of the correct data format.
+
```

Comment:
Please unify these new lines for consistency. Right now some of them has new line and some have missing. Same for line 82 and 88.

Reply:
done